README.md

```diff
@@ -71,18 +71,18 @@ import (
 )
 
 func main() {
-    // create new client with default option for server url authenticate by token
+    // Create new client with default option for base server url authenticated by token
```

Comment:
there is no default option for base server URL,  the following comment would probably look better (all files):
```suggestion
    // Create a new client using an InfluxDB base URL and an authentication token
```